test/swim_test.js

```diff
@@ -22,25 +22,13 @@ var mock = require('./mock');
 var test = require('tape');
```

Comment:
These changes are all incidental. Feel free to ignore them. They have nothing to do with `request-proxy-send` introduction.